logback-ecs-encoder/src/main/java/co/elastic/logging/logback/EcsEncoder.java

```diff
@@ -198,4 +208,11 @@ public void setEventDataset(String eventDataset) {
     public void setThrowableConverter(ThrowableHandlingConverter throwableConverter) {
         this.throwableConverter = throwableConverter;
     }
+    public Layout<ILoggingEvent> getLayout() {
+        return this.layout;
+    }
+
+    public void setLayout(Layout<ILoggingEvent> layout) {
+        this.layout = layout;
+    }
```

Comment:
```suggestion
    public void setMessageLayout(Layout<ILoggingEvent> messageLayout) {
        this.messageLayout = messageLayout;
    }
```
Add a javadoc saying this `Layout` will be applied specifically to format the `message` field based on the logging event.